troupon/deals/views.py

```diff
@@ -270,3 +270,23 @@ def get(self, *args, **kwargs):
             })
 
         return HttpResponse(template.render(context))
+
+class AboutView(View):
```

Comment:
**Code Quality: Nitpick**

Replace all of these with `TemplateView` to avoid repetition of `render` command.